src/utils/menu.js

```diff
@@ -1,52 +1,85 @@
 const menu = [
   {
-    name: 'Apps',
+    name: '1hive Products',
     sublinks: [
       {
-        name: 'Honey Faucet',
-        link: 'https://faucet.1hive.org/',
-        description: 'Register with BrightId to claim Honey'
+        name: 'Honeypot',
+        link: 'https://1hive.org',
+        description: 'DAO Governance'// - Create and stake on proposals'
       },
       {
-        name: '1hive DAO Interface',
-        link: 'https://1hive.org',
-        description: 'Create Proposals and Stake Honey'
+        name: 'Honey Faucet',
+        link: 'https://faucet.1hive.org/',
+        description: 'Claim Free Honey'// - Register with BrightId to claim Honey'
       },
       {
-        name: 'Honeyswap Interface',
+        name: 'Honeyswap',
         link: 'https://honeyswap.org/',
-        description: 'Swap tokens and supply liquidity'
+        description: 'Decentralized Exchange' //Swap & Supply Liquidity - Swap tokens and supply liquidity. Create new pools'
       },
       {
-        name: 'Honeyswap Analytics',
-        link: 'https://info.honeyswap.org',
-        description: 'Honeyswap analytics and historical data'
+        name: 'Honeycomb',
+        link: 'https://hny.farm/',
+        description: 'Yield Farming'// - Stake Honeyswap LP tokens to claim Honey.'
       }
     ]
   },
   {
-    name: 'Documentation',
+    name: 'External Apps',
     sublinks: [
       {
-        name: '1hive Handbook',
-        link: '/docs/dao',
-        description: 'Learn about 1hive and how to contribute'
+        name: 'BrightID',
+        link: 'https://www.brightid.org/',
+        description: 'Verify for Faucet'
```

Comment:
I would find a better tagline here, since it's not only for the faucet once Celeste launches.

Reply:
Faucet being so short term can we just call it Verify for Celeste until we need it for something else.  

I am afraid to call it something inclusive like "universal proof of uniqueness" as it doesn't make any sense to the user. they can read up on the details but these links need to connect them to something quickly and easily without struggle. I am open to suggestions. I will ping the discord as well.